root/recording/RecordingIndex.js

```diff
@@ -50,6 +50,7 @@ const RecordingAppearancesTable = ({
         <th>{l('Track Artist')}</th>
         <th>{l('Release Title')}</th>
         <th>{l('Release Artist')}</th>
+        <th>{l('Release Type')}</th>
```

Comment:
Wouldn't "Release Group Type" be better?

Reply:
It would, yes :)